articles/install-guide/index.md

```diff
@@ -1,40 +1,89 @@
 ---
-title: Setting up the Q# development environment 
-author: QuantumWriter
-ms.author: Alan.Geller@microsoft.com 
-ms.date: 12/11/2017
+title: Getting Started with the Microsoft Quantum Development Kit
+author: anpaz-msft
+ms.author: andres.paz@microsoft.com 
+ms.date: 3/26/2019
 ms.topic: article
 uid: microsoft.quantum.install
 ---
 
-# Getting Started with the Quantum Development Kit
+# Getting Started with the Microsoft Quantum Development Kit #
 
-The Quantum Development Kit can be used on a variety of platforms (Windows, Linux, Mac) and with a variety of development environments (Visual Studio, Visual Studio Code, or the command line).  Additionally, Q# quantum programs are executed on one of many target machines provided by the Quantum Development Kit (e.g., full-state simulator, trace simulator or resource estimator).  These target machines can be invoked from any .NET host language, or from Python.   
+A great way to get started with the Quantum Development Kit is via some of the Jupyter notebooks available in our [GitHub Quantum repository](https://github.com/Microsoft/Quantum.git). 
+These notebooks can be launched directly in a web browser via [binder](https://mybinder.org/). In particular:
 
-The Quantum Development Kit can be used either with the Visual Studio 2017 integrated development environment, or a development editor such as Visual Studio Code.
-Alternatively, the Quantum Development Kit can be used with the [command line directly](xref:microsoft.quantum.install.cmd-line).  A separate section gives additional information on how to integrate with [Python](xref:microsoft.quantum.install.python).
+* **[Q# Notebooks](https://mybinder.org/v2/gh/Microsoft/Quantum.git/master?filepath=Samples%2Fsrc%2FIntroToIQSharp%2FNotebook.ipynb)**: explains how to compile and simulate Q# operations inside a Jupyter notebook.
+* **[Teleport](https://mybinder.org/v2/gh/Microsoft/Quantum.git/master?filepath=Samples%2Fsrc%2FTeleportation%2FNotebook.ipynb)**: shows how to implement the teleport algorithm using Q#.
 
-<table>
-<tr>
-<td>
-[<img src="../media/install-guide-select-vs-2017.png" width="90%">](xref:microsoft.quantum.install.vs-2017)
-<div class="nextstepaction">
-[Install on Visual Studio 2017 (Windows 10)](xref:microsoft.quantum.install.vs-2017)
-</div>
-</td>
+Be mindful that all changes you do to notebooks launched this way cannot be saved. In order 
+to create new notebooks or write new quantum applications you will need to install 
+IQ# to your local machine.
```

Comment:
the IQ# Jupyter Kernel

Reply:
`(xref)` works if viewed from the preview site (i.e. https://review.docs.microsoft.com/en-us/quantum/install-guide/csharp?view=qsharp-preview&branch=anpaz%2Fgetting-started), but don't work when using markdown preview from github.